src/NzbDrone.Common/Instrumentation/Sentry/SentryTarget.cs

```diff
@@ -172,9 +203,32 @@ protected override void Write(LogEventInfo logEvent)
                     return;
                 }
 
+                // bodge to try to get the exception message in English
+                // https://stackoverflow.com/questions/209133/exception-messages-in-english
+                // There may still be some localization but this is better than nothing.
+                string message = string.Empty;
+
+                // only try to use the exeception message to fingerprint if there's no inner
+                // exception and the message is short, otherwise we're in danger of getting a
+                // stacktrace which will break the grouping
+                if (logEvent.Exception != null && logEvent.Exception.InnerException == null)
+                {
+                    var t = new Thread(() => {
+                            message = logEvent.Exception?.Message;
+                        });
+                    t.CurrentCulture = CultureInfo.InvariantCulture;
+                    t.CurrentUICulture = CultureInfo.InvariantCulture;
+                    t.Start();
+                    t.Join();
+                }
+
+                Console.WriteLine($"Sentry fingerprint message {message}");
+
                 if (logEvent.Exception != null)
                 {
                     sentryEvent.Fingerprint.Add(logEvent.Exception.GetType().FullName);
+                    sentryEvent.Fingerprint.Add(logEvent.Exception.TargetSite.ToString());
+                    sentryEvent.Fingerprint.Add(message.Length < 200 ? message : string.Empty);
```

Comment:
message is already added as a fingerprint on line 194, does this give us anything more?

Reply:
On line 194, the message in the `LogEvent` is added.  This is the message in the actual exception and should be the thing that stops all the sqlite exceptions being grouped

Reply:
Ok, must have been the exception I was testing we were using the message for log message so they were the same